eva/parser/delete_statement.py

```diff
@@ -0,0 +1,103 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List
+
+from eva.expression.abstract_expression import AbstractExpression
+from eva.expression.constant_value_expression import ConstantValueExpression
+from eva.parser.statement import AbstractStatement
+from eva.parser.table_ref import TableRef
+from eva.parser.types import StatementType
+
+
+class DeleteTableStatement(AbstractStatement):
+    """
+    Delete Table Statement constructed after parsing the input query
+
+    Attributes
```

Comment:
We don't use this style of the docstring. Please modify it. Thanks!

Reply:
Is it better now